recipes/norbert/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "norbert" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - git_url: https://github.com/sigsep/norbert.git
```

Comment:
You should use the tarballs from PyPI or GitHub for the reasons listed here:

https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos

Reply:
The reason I choosed `git` url instead of *PyPi* tarball archive is that I am not the owner of this project (as well as for `musdb`), and the packaged version do not embed the LICENSE file, so the CI would fail :/

What would be the best option then ?

Reply:
If you include the licence file in the recipe directory next to the `meta.yaml` file conda will find it from there.

Reply:
> If you include the licence file in the recipe directory next to the `meta.yaml` file conda will find it from there.

Magic. Let's go for it then !